.github/workflows/push.yml

```diff
@@ -42,7 +42,6 @@ jobs:
           cache: 'npm'
           node-version-file: '.nvmrc'
       - run: npm ci
-      - run: 'npm run lint -- --rule "linebreak-style: off"'
```

Comment:
I don't want to disable linting _entirely_ in CI on Windows, otherwise we don't find out if it regresses; I'm hoping that the rule exclusion is no longer needed - so `- run: npm run lint` would work.

Reply:
mb, I didn't realise I removed the entire line, not just the rule. Added it back

Reply:
🎲 No dice, sadly: https://github.com/textbook/starter-kit/actions/runs/4809758053/jobs/8561580638. It [looks like](https://stackoverflow.com/q/2517190/3001761) it might be quite complicated to get this to work consistently!

Reply:
I did find this, the `* text eol=lf` might be the one to work?
https://github.com/actions/checkout/issues/135#issuecomment-613361104